src/autonlp/evaluate.py

```diff
@@ -0,0 +1,77 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Tuple
+
+from .tasks import TASKS
+from .utils import BOLD_TAG, GREEN_TAG, RESET_TAG, YELLOW_TAG
+
+
+JOB_STATUS = (
+    ("⌚", "queued"),
+    ("🚀", "start"),
+    ("⚙", "data_munging"),
+    ("🏃", "model_evaluating"),
+    ("✅", "success"),
+    ("❌", "failed"),
+)
```

Comment:
Let's share this between training and eval jobs

Reply:
I think it makes sense to move the Evaluate class to the `model.py` file

Reply:
removed for time being